src/apis/roadmap/index.ts

```diff
@@ -1,19 +1,19 @@
 import axios from 'axios';
 import {PaginatedResponse} from 'types/api';
-import {RoadmapTask} from 'types/roadmap';
```

Comment:
I don't think we should change the roadmap `Task` to `Bounty` as `Bounty` is offered to external contributors, while roadmap are the tasks for the core team members

Reply:
done!